static/src/stylesheets/module/commercial/glabs/_hosted.scss

```diff
@@ -1,13 +1,17 @@
 $hosted-stripe: 28px;
+$hosted-video-width: 960px;
+$hosted-video-height: 540px;
 
 @mixin hosted-fade-in {
+    visibility: visible;
     opacity: 1;
-    transition: opacity 1s linear;
+    transition: opacity 1s linear, visibility 0s;
```

Comment:
It depends on the design of your page. Screen readers won't - unless coaxed otherwise - tell users that a piece of content has been removed from the page. It looks as though we're using this mixin on the video player's controls. If I'm using a screen reader, start a video, and then need to stop it, will I still be able to operate those controls in the way I originally would have?